ui/src/testing/stubs/profile-service-stub.ts

```diff
@@ -0,0 +1,65 @@
+import {Profile} from 'generated';
+import {Observable} from 'rxjs/Observable';
+
+class ProfileStub implements Profile {
```

Comment:
Could we just use the Profile class directly? I don't understand what ProfileStub is adding. For example, could there be a static Profile here with default values, and ProfileStubService just serves that up (or, if Profile is not immutable, serves up a copy for each request)?

Reply:
I believe using the profile directly should be fine. I can't think of a reason it wouldn't be. I was following the pattern of the workspace stub and cohort stub, which I believe we used because of optional fields/creation, but should not be necessary here?